great_expectations/profiler/parameter_builder/metric_parameter_builder.py

```diff
@@ -34,27 +47,35 @@ def _build_parameters(
         :return: a dictionary of format {'parameters': A given resolved metric}
         """
         # Obtaining any necessary domain kwargs from rule state, otherwise using instance var
-        if self._metric_domain_kwargs.startswith("$"):
-            metric_domain_kwargs = rule_state.get_value(self._metric_domain_kwargs)
+        if isinstance(
```

Comment:
❤️ Love this defensive check

Reply:
I think that the credit goes to @jcampbell :-)

Reply:
Oh wait...  No, that one was me! :)